### ISSUE TITLE
adding backend functionality for signup and login pages.

### ISSUE THREAD

**Issue by T-Rahul-prabhu-38** [Mohit5Upadhyay/realTimeTrackingApplication#44]
even after submitting the form ,it doesnt save the data.
@Mohit5Upadhyay if you assign me this i can add a feature to submit the data and store in local storage.

<img width="1389" alt="Screenshot 2024-10-11 at 11 48 10 PM" src="https://github.com/user-attachments/assets/92e02be0-1029-400b-b3ea-bcfc767e0cac">

**Comment by Mohit5Upadhyay** [comment]
Yes @T-Rahul-prabhu-38  go ahead to implement the functionality.

**Comment by muskiagrwal** [comment]
i want to work on this issue .assign me this project
